src/sidebar/components/Annotation/Annotation.tsx

```diff
@@ -105,11 +105,14 @@ function Annotation({
   const annotationDescription = isSaved(annotation)
     ? annotationRole(annotation)
     : `New ${annotationRole(annotation).toLowerCase()}`;
+  const suffix = store.isAnnotationHighlighted(annotation)
+    ? ' - Highlighted'
+    : '';
```

Comment:
I'm not sure if just adding this suffix to the accessible label is enough to mark it as highlighted. Perhaps the word "highlighted" is not clear enough with the context it has here.

Reply:
I think this is fine, as an audible representation of what we are showing visually. I would suggest the name `state` rather than `suffix`, to make it a bit clearer what the suffix is used for.